routes/writing.js

```diff
@@ -105,6 +133,17 @@ router.post("/edit/:id", function (req, res) {
   });
 });
 
+
+router.get("/posterror", function (req, res) {
```

Comment:
Don't write a separate route just for errors. Errors should be written for each individual route.